doc/source/structures/volumes_and_files/volume.rst

```diff
@@ -34,27 +34,31 @@ Represents a :struct:`kOSProcessor` hard disk or the archive.
 
         * - :attr:`FILES`
           - :struct:`Lexicon`
-          - Lexicon of all files on the volume
+          - Lexicon of all files and directories on the volume
 
         * - :attr:`POWERREQUIREMENT`
           - :ref:`scalar <scalar>`
           - Amount of power consumed when this volume is set as the current volume
```

Comment:
This is no longer true... ish.  I think I broke this actually when I changed the power logic.  We do have an option to have EC draw based on file system size, however it is defaulted to zero (and I don't anticipate us changing that).  But we still call the method in `ProcessElectricity`.  I'm gonna have to work on that logic anyways for #1363, so this change will flow into that fix easily.